### DISCUSSION TITLE
Battery health and buzzing noise

### DISCUSSION THREAD

**Discussion by Reach-Orange-CN** [seerge/g-helper#2964] [Q&A] [answered]
### ROG Zephyrus G16 GU605MV, have been using for 5 months
**1.** I've been using my computer with the power connected all the time, but once I used it on battery, and it had about 40% charge left. The next day, when I restarted the computer, the **battery health** dropped from **100% to 96.5%**. I am wondering if the bettery health can be recovered or it is just the problem of display.
**2.** My computer makes a **faint buzzing noise** that can be heard when I put my ear close to the keyboard surface. I can't remember if this issue was present before or did it only start occurring after I updated the **BIOS**?

**Comment by seerge** [comment, marked as answer]
@Reach-Orange-CN 

1. Windows / system constantly recalibrates battery health, this is a very approximate measurement. It can go up and down.
2. It's called "coil whine"

**Reply by Reach-Orange-CN** [reply]
thank you so much!
